cloudsim-plus/src/main/java/org/cloudbus/cloudsim/brokers/DatacenterBrokerNull.java

```diff
@@ -56,6 +56,7 @@
     @Override public List<Cloudlet> getCloudletSubmittedList() { return Collections.emptyList(); }
     @Override public void setVmComparator(Comparator<Vm> comparator) {/**/}
     @Override public void setCloudletComparator(Comparator<Cloudlet> comparator) {/**/}
+    @Override public Vm defaultVmMapper(Cloudlet cloudlet) { return Vm.NULL; }
     @Override public Function<Cloudlet, Vm> getVmMapper() { return c -> Vm.NULL; }
```

Comment:
getVmMapper() is not needed anymore and should be removed from the DatacenterBroker interface too.

Reply:
Since, there was a setVmMapper() method, I thought there should also be a getVmMapper() for completeness. I can remove it, if we don't see any use-case for that method though.

Reply:
Ok.

Reply:
So we keep the `getVmMapper()` method, right?